apps/src/templates/manageStudents/DownloadParentLetters.jsx

```diff
@@ -17,31 +17,33 @@ export default class DownloadParentLetters extends Component {
     sectionId: PropTypes.number
```

Comment:
can remove the numStudents prop now too.

Reply:
good catch! thanks!